README.adoc

```diff
@@ -1,38 +1,32 @@
-= Address Book (Level 4)
+= Health Book
+
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/nusCS2113-AY1819S1/addressbook-level4[image:https://travis-ci.org/nusCS2113-AY1819S1/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
-https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
+https://travis-ci.org/CS2113-AY1819S1-T12-2/main[image:https://travis-ci.org/CS2113-AY1819S1-T12-2/main.svg?branch=master[Build Status]]
+https://app.codacy.com/app/xhxh96/main?utm_source=github.com&utm_medium=referral&utm_content=CS2113-AY1819S1-T12-2/main&utm_campaign=Badge_Grade_Dashboard[image:https://api.codacy.com/project/badge/Grade/b3b9161e1b4a424aa5d1da1a6e759be6[Codacy Badge]]
```

Comment:
Good job for updating travis and codacy badges, will be good to add coveralls and appveyor badges later (: